documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -0,0 +1,477 @@
+      "pattern": "inventory.inventory.products"
+    },
+    "negate": false
+  }
+}
+----
+==== Edit
+To edit a transform, go to the `Transform` menu and then click the `action` menu of the destination you want to edit, then click `Edit`.
+
+[NOTE]
+Editing a transform will affect all pipelines that use it.
+
+==== Delete
+To delete a transform, go to the `Transform` menu and then click the `action` menu of the transform you want to delete, then click `Delete`.
+A transform can be deleted only if it is not used in any pipeline; otherwise, you will receive an error.
+When the transform is no longer used in any pipeline, you can delete it using the `Delete` button.
+
+=== Pipeline
+The pipeline section is the place where you connect the "dots". You can define where your data comes, how to eventually transform them and where they should go.
+
+[.responsive]
+video::cY7TS0nWBoY[youtube, title="Create, edit and remove a pipeline"]
+
+==== Create
+In the pipeline menu you can click on `Create your first pipeline` and you will get into the `Pipeline Designer`.
+Here you can add the pieces that composes you data pipeline. First of all, you need to add a source clicking on the `+ Source` box and the you can either choose a previously created source or directly create a new one.
+
+Similarly, you can add a destination clicking on the `+ Destination` box.
+
+If you want to apply some transformation to your data, you can add it in the same way just clicking on the `+ Transform` box.
+
+When a transform as a predicate configured, you will see a image:predicate-icon.png[alt text] on top of it. A tooltip will show the name of the predicate used.
+
+Once finished designing your pipeline you can click on `Configure Pipeline` and then you can now configure the name, the description and the logging level.
+
+==== Delete
+To delete a pipeline, go to the `Pipeline` menu and then click the `action` menu of the pipeline you want to delete, then click `Delete`.
+Only the pipeline will be removed, the source, the destination and the transformations will not be deleted.
+
+==== Edit
+To edit a pipeline, go to the `Pipeline` menu and then click the `action` menu of the destination you want to edit, then click `Edit pipeline`.
+As first step you can modify the transformations through the pipeline designer, we will go deeper in this part in the next section, and then you can edit the name, the description and the log level.
+
+===== Remove or ordering transformations
+Once you are in the `Pipeline designer` you can modify the order of transformations or delete one by clicking on the image:transformation-box-edit.png[alt text] icon.
+
+==== Observability
+Observability is currently limited to viewing the Debezium Server logs.
+You can go to `Pipeline` then click on the pipeline name of your interest and then click on the `Pipeline logs`.
+The other ways is to go to `Pipeline` and then click the `action` menu of the pipeline you are interested, then click `View logs`.
```

Comment:
```suggestion

```
I'd suggest removing the alternative method on the principle that it's better for the documentation to be opinionated, and leave it to users to discover shortcuts and alternative methods that are available through mechanisms that we can assume the user is familiar with.